ts/parts/Pointer.ts

```diff
@@ -1716,6 +1725,75 @@ Highcharts.Pointer.prototype = {
 
     },
 
+    /**
+     * @private
+     * @param {Highcharts.PointerEventObject} e
+     * @return {boolean}
+     */
+    isStickyTooltip: function (this: Highcharts.Pointer, e: Highcharts.PointerEventObject): boolean {
+        const chart = this.chart;
+        const point = chart.hoverPoint;
+        const tooltip = chart.tooltip;
+        const eventPosition: Highcharts.PositionObject = {
+            x: e.chartX,
+            y: e.chartY
+        };
+
+        const absoluteBounding = function (element: Highcharts.SVGElement): Highcharts.BBoxObject {
```

Comment:
Instead of the new `absoluteBounding` function , try using the existing `Highcharts.offset` function. It reports the position relative to the _page_, but then `chart.pointer.chartPosition` can be substracted.

Reply:
This should be solved.